### ISSUE TITLE
Latte dock

### ISSUE THREAD

**Issue by themrdanielvp** [feren-OS/Bug-Reporting-Center#295]
Hi.
Since Latte Dock is getting more and more buggy after each version of Plasma and since it is no longer maintaned are there any plans to swap it out for a different dock?

**Comment by dominichayesferen** [comment]
Yes there are plans. They're currently held up by https://bugs.kde.org/show_bug.cgi?id=457498
